README.md

```diff
@@ -176,273 +86,176 @@ You can view the configurations you've loaded with the `configurations list` com
-┃ Host Name ┃ GraphQL API URL                  ┃ Aerie Gateway URL     ┃ Authentication Method ┃ Username ┃
-┡━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━┩
-│ localhost │ http://localhost:8080/v1/graphql │ http://localhost:9000 │ None                  │          │
-└───────────┴──────────────────────────────────┴───────────────────────┴───────────────────────┴──────────┘
-```
-
-#### Via the CLI
-
-If you haven't been provided a JSON configuration for a host, you can create a configuration by running `aerie-cli configurations create` and follow the prompts. For example, to configure localhost:
-
-```sh
-➜  aerie-cli configurations create
-   Name: localhost
-   GraphQL URL: http://localhost:8080/v1/graphql
-   Gateway URL: http://localhost:9000
-   Authentication method (None, Native, Cookie) [None]: None
+┏━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┓
+┃ Host Name ┃ GraphQL API URL                  ┃ Aerie Gateway URL     ┃ Username ┃
+┡━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━┩
+│ localhost │ http://localhost:8080/v1/graphql │ http://localhost:9000 │          │
+└───────────┴──────────────────────────────────┴───────────────────────┴──────────┘
 ```
 
-To view available configurations, use `aerie-cli configurations list`:
-
-```sh
-➜  aerie-cli configurations list
-Configuration file location: /Users/<username>/Library/Application Support/aerie_cli/config.json
-
-                                         Aerie Host Configurations
-┏━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┓
-┃ Host Name ┃ GraphQL API URL                  ┃ Aerie Gateway URL     ┃ Authentication Method ┃ Username ┃
-┡━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━┩
-│ localhost │ http://localhost:8080/v1/graphql │ http://localhost:9000 │ None                  │          │
-└───────────┴──────────────────────────────────┴───────────────────────┴───────────────────────┴──────────┘
-```
-
-#### Handling Authentication
-
-Each configuration must specify an authentication method, which depends on how the Aerie instance is deployed. Aerie CLI provides three ways to authenticate with an Aerie host:
+#### Defining hosts via the CLI
 
-| Method   | Description                                                     |
-| :------- | :-------------------------------------------------------------- |
-| `None`   | No authentication (e.g., for a local deployment)                |
-| `Native` | Default Aerie authentication scheme (Token-based)               |
-| `Cookie` | Cookie-based authentication (used on some advanced deployments) |
+If you haven't been provided a JSON configuration for a host, you can create a configuration by running `aerie-cli configurations create` and follow the prompts. 
```

Comment:
Minor: This section used to have an example of what the prompts were. It may be worth reincluding here.

Additionally, when I used the `configurations create` command, I was unable to create a configuration that used external auth. Is this currently possible?

Reply:
I was trying to minimize the number of examples in the README that just show a command in use (less documentation to maintain if the prompts change), but I could re-add if you think that's useful.

I didn't create a way to define external auth because I feel like it's complicated enough that folks are more likely to load an existing JSON than to labor through the prompts, but I could be wrong about that?

Reply:
Just added a table that describes the main fields of a host configuration.